From 693488ff0c72e5258eb7a33befb027d48a7517fe Mon Sep 17 00:00:00 2001
From: Claudia Pellegrino <claui@users.noreply.github.com>
Date: Thu, 4 Sep 2025 13:17:35 +0200
Subject: [PATCH 1/5] Add resource for sexual assault help in DE

The helpline, including the chat, is operated by N.I.N.A. e.V. [1] and
overseen by the Independent Federal Commissioner Against Child Sexual
Abuse. [2]

[1]: https://nina-info.de

[2]: https://www.hilfe-portal-missbrauch.de/en/helpline
---
 data/resources.json | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/data/resources.json b/data/resources.json
index 6962f56..2fce02d 100644
--- a/data/resources.json
+++ b/data/resources.json
@@ -318,6 +318,14 @@
             "number": "08000 116 016",
             "description": "24/7 violence against women helpline"
           }
+        ],
+        "sexual_assault": [
+          {
+            "name": "Hilfe-Telefon Sexueller Missbrauch (Youth)",
+            "number": "0800 22 55 530",
+            "description": "Mon, Wed, Fri: 09:00–14:00; Tues, Thurs: 15:00–20:00",
+            "website": "https://www.hilfe-portal-missbrauch.de"
+          }
         ]
       }
     },

From 6645eef054da018301bd97945dd75dd29157ac47 Mon Sep 17 00:00:00 2001
From: Claudia Pellegrino <claui@users.noreply.github.com>
Date: Thu, 4 Sep 2025 13:18:02 +0200
Subject: [PATCH 2/5] Add resources for substance abuse help in DE
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

All helplines are overseen by the Federal Institute of Public Health
(BIÖG). [1]

Gambling addiction is not strictly substance abuse; add it anyway
because it’s good enough under this section as long as there’s no other
section that fits better.

[1]: https://www.bioeg.de/service/infotelefone/
---
 data/resources.json | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/data/resources.json b/data/resources.json
index 2fce02d..d77e2d5 100644
--- a/data/resources.json
+++ b/data/resources.json
@@ -326,6 +326,32 @@
             "description": "Mon, Wed, Fri: 09:00–14:00; Tues, Thurs: 15:00–20:00",
             "website": "https://www.hilfe-portal-missbrauch.de"
           }
+        ],
+        "substance_abuse": [
+          {
+            "name": "Sucht- und Drogen-Hotline",
+            "number": "01806 313031",
+            "description": "08:00–24:00 (0,20 € per call)",
+            "website": "https://www.bioeg.de/service/infotelefone/sucht-drogen-hotline/"
+          },
+          {
+            "name": "Infotelefon zur Suchtvorbeugung",
+            "number": "0221 89 20 31",
+            "description": "Mon–Thu: 10:00–22:00; Fri–Sun: 10:00–18:00",
+            "website": "https://www.bioeg.de/service/infotelefone/suchtvorbeugung/"
+          },
+          {
+            "name": "Telefonberatung zur Rauchentwöhnung",
+            "number": "0800 8 31 31 31",
+            "description": "Mon–Thu: 10:00–22:00; Fri–Sun: 10:00–18:00",
+            "website": "https://www.bioeg.de/service/infotelefone/rauchentwoehnung/"
+          },
+          {
+            "name": "Telefonberatung zur Glücksspielsucht",
+            "number": "0800 1 37 27 00",
+            "description": "Mon–Thu: 10:00–22:00; Fri–Sun: 10:00–18:00",
+            "website": "https://www.bioeg.de/service/infotelefone/gluecksspielsucht/"
+          }
         ]
       }
     },

From ca9325ded37f9d12c03bb897fc21e5ed46f5225b Mon Sep 17 00:00:00 2001
From: Claudia Pellegrino <claui@users.noreply.github.com>
Date: Thu, 4 Sep 2025 13:25:49 +0200
Subject: [PATCH 3/5] Add resource for child abuse help in DE

This helpline is fostered by the Federal Ministry for Education, Family
Affairs, Senior Citizens, Women and Youth. [1]

[1]: https://kinderschutzhotline.de/
---
 data/resources.json | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/data/resources.json b/data/resources.json
index d77e2d5..0ea4a75 100644
--- a/data/resources.json
+++ b/data/resources.json
@@ -319,6 +319,14 @@
             "description": "24/7 violence against women helpline"
           }
         ],
+        "child_abuse": [
+          {
+            "name": "Medizinische Kinderschutzhotline",
+            "number": "0800 19 210 00",
+            "description": "24/7 child abuse helpline",
+            "website": "https://kinderschutzhotline.de"
+          }
+        ],
         "sexual_assault": [
           {
             "name": "Hilfe-Telefon Sexueller Missbrauch (Youth)",

From 7170e830e987362708e938c040636c851104f365 Mon Sep 17 00:00:00 2001
From: Claudia Pellegrino <claui@users.noreply.github.com>
Date: Thu, 4 Sep 2025 13:31:54 +0200
Subject: [PATCH 4/5] Add resource for veterans support in DE

This helpline is operated by the Bund Deutscher EinsatzVeteranen e. V.
(German Veterans Association). [1] [2]

[1]: https://www.veteranenverband.de/hilfe/

[2]: https://de.wikipedia.org/wiki/Bund_Deutscher_EinsatzVeteranen
---
 data/resources.json | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/data/resources.json b/data/resources.json
index 0ea4a75..3539514 100644
--- a/data/resources.json
+++ b/data/resources.json
@@ -335,6 +335,14 @@
             "website": "https://www.hilfe-portal-missbrauch.de"
           }
         ],
+        "veterans": [
+          {
+            "name": "Hilfeabteilung des Bundes Deutscher EinsatzVeteranen e.V.",
+            "number": "030 209 242 08",
+            "description": "Mon–Fri 08:00–17:00",
+            "website": "https://www.veteranenverband.de/hilfe"
+          }
+        ],
         "substance_abuse": [
           {
             "name": "Sucht- und Drogen-Hotline",

From f6c72e38db3eeda32330cead0bacda6e9053405e Mon Sep 17 00:00:00 2001
From: Claudia Pellegrino <claui@users.noreply.github.com>
Date: Thu, 4 Sep 2025 13:47:26 +0200
Subject: [PATCH 5/5] Add resource for LGBTQ support in DE

In Germany, most LGBTQ helplines are operated locally, but this is the
only one I could find that offers support on a federal level.
It is operated by the Federal Anti-Discriminiation Agency. [1]

[1]: https://www.antidiskriminierungsstelle.de/EN
---
 data/resources.json | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/data/resources.json b/data/resources.json
index 3539514..7f969a2 100644
--- a/data/resources.json
+++ b/data/resources.json
@@ -343,6 +343,14 @@
             "website": "https://www.veteranenverband.de/hilfe"
           }
         ],
+        "lgbtq": [
+          {
+            "name": "Antidiskriminierungsstelle des Bundes",
+            "number": "0800 546 546 5",
+            "description": "Mon–Thu: 09:00–15:00",
+            "website": "https://www.antidiskriminierungsstelle.de/DE/wir-beraten-sie/wir-beraten-sie-node.html"
+          }
+        ],
         "substance_abuse": [
           {
             "name": "Sucht- und Drogen-Hotline",
